ui/Dockerfile.dev

```diff
@@ -0,0 +1,24 @@
+FROM node:alpine3.16
+
+# Set working DIE
+WORKDIR /home/app/ui
+
+# Upgrade Alpine Packages
+RUN apk -U upgrade
+
+# Add bash to run tests and other scripts
+RUN apk add --no-cache bash
+
+# Copy package.json and package-lock.json
+COPY package*.json .
+
+# Install node packages
+RUN npm install
```

Comment:
This should be `npm ci` as we don't want to have different npm package version than what we have in package-lock.json.